packages/components/src/visually-hidden/index.js

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
This empty space is not necessary. Why is ESLint not complaining? :) We really need Prettier 😍

Reply:
Funny because the space should be below, newline at the end.